interbtc-spec/docs/source/spec/vault-registry.rst

```diff
@@ -749,12 +759,16 @@ Specification
 
 *Postconditions*
 
-* If ``oldVault`` is *not* liquidated:
-   * The ``oldVault``'s ``toBeRedeemedTokens`` and ``issuedTokens`` MUST be decreased by the amount ``tokens``.
-   * The ``oldVault``'s collateral free balance MUST be increased by ``tokens / toBeRedeemed``.
-* If ``oldVault`` *is* liquidated, the liquidation vault's ``toBeRedeemedTokens`` and ``issuedTokens`` are decrease by the amount ``tokens``.
-* If ``newVault`` is *not* liquidated, its ``toBeIssuedTokens`` is decreased by ``tokens``, while its ``issuedTokens`` is increased by the same amount.
-* If ``newVault`` *is* liquidated, the liquidation vault's  ``toBeIssuedTokens`` is decreased by ``tokens``, while its ``issuedTokens`` is increased by the same amount.
+* If the ``oldVault`` *is* liquidated:
+
+   * The amount ``toBeReleased`` is calculated as ``(oldVault.liquidatedCollateral * tokens) / toBeRedeemedTokens``.
+   * The ``oldVault``'s ``liquidatedCollateral`` MUST decrease by ``toBeReleased``.
+   * Function :ref:`reward_depositStake` MUST complete successfully - parameterized by ``oldVault`` and ``toBeReleased``.
```

Comment:
```suggestion
   * Function :ref:`staking_depositStake` MUST complete successfully - parameterized by ``oldVault``, ``oldVault`` and ``toBeReleased``.
```

Reply:
Same as above, this affects interBTC reward stake.